examples/pytorch/lda/lda_model.py

```diff
@@ -0,0 +1,229 @@
+# Copyright 2021 Yifei Ma
+# with references from "sklearn.decomposition.LatentDirichletAllocation"
+# with the following original authors:
+# * Chyi-Kwei Yau (the said scikit-learn implementation)
+# * Matthew D. Hoffman (original onlineldavb implementation)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os, functools, warnings
+import numpy as np, scipy as sp
+import torch
+import dgl
+from dgl import function as fn
+
+
+def _lbeta(alpha, axis):
+    return torch.lgamma(alpha).sum(axis) - torch.lgamma(alpha.sum(axis))
+
+
+_sklearn_random_init = torch.distributions.gamma.Gamma(100, 0.01)
```

Comment:
Why is it named this way?

Reply:
This works better than uniform init. I tried uniform init, but it could not match the sklearn numbers.

Reply:
Interesting.

Reply:
I actually found a bug here - torch uses rate=1/scale parameter. Committing a change.